### ISSUE TITLE
static release binaries are zipped twice

### ISSUE THREAD

**Issue by JanMa** [openbao/openbao#273]
The static binaries which are built for the initial Alpha release are all zipped twice. This means you also have to unzip them twice to get to the `bao` binary. I am sure this is not intentional so we should fix it.

**Comment by cipherboy** [comment]
@JanMa Yeah, its an artifact of directly re-uploading the artifacts from the release pipeline... As [I mentioned](https://github.com/openbao/openbao/issues/261#issuecomment-2034374767) on #261, the GH Actions artifact uploader automatically zips everything, which creates double-zips in most cases (except where tar was used and it was simply bundled and not compressed, but `.tar.zip` is a weird combination!).

I think switching to the new goreleaser for GA/alpha 2 will fix this based on the testing artifacts that @naphelps had created. :D 

Upstream had internal tooling that pulled down the release artifacts, unzipped them, did the signing, and then uploaded to the release channels.

**Comment by naphelps** [comment]
Fixed the double zip.
